### DISCUSSION TITLE
Build a Web portal to showcase Nepal's best features

### DISCUSSION THREAD

**Discussion by gorkhadev** [TourismKit/beautifulnepal.org#4] [Ideas]
We need to make a portal to display all the best places, culture, cities and others in Nepal

**Comment by devsdenepal** [comment]
We can use json data for storing and fetching data for places,culture, cities,etc as you said.
![image](https://github.com/beautifulnepal/beautifulnepal.org/assets/111997815/41c2dd7e-0ae4-447e-8011-c8ad1740ab88)
Additionaly, we can also use open source data.
